### PR TITLE
Add Rosello file to class repo

### PR DESCRIPTION
**Pull request by CarlosVendo**
- @Albertovs  show me some of your great images that you store on your laptop 😄 🙏 

- where are you guys from?

### CODE REVIEW

File: Albertovs-BCN.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Barcelona, Spaim
+
+### Restaurants
+
+- Can Punyetes
+- Porsant
```
Comment (clangue) [inline Albertovs-BCN.md]: @CarlosVendo funny enough I put the same

File: Albertovs-BCN.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+### Restaurants
+
+- Can Punyetes
+- Porsant 
+
+### Attractions for Tourists
+
+- Gothic 
+- Born
```
Comment (clangue) [inline Albertovs-BCN.md]: @CarlosVendo --> WE DONT WANT GUIRIES THERE!!!!

File: Albertovs-BCN.md (hunk 1 above)
Comment (CarlosVendo) [inline Albertovs-BCN.md]: jaja @clangue  it was Alberto on my issue!

**Comment by CarlosVendo** [comment]
👍

**Pull request merged**